features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2557,6 +2557,31 @@
             <Scopes>internal_login</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/organizations" secured="true" http-method="GET">
```

Comment:
Lets remove the empty new line between the configurations

Reply:
Related to the reply above https://github.com/wso2/carbon-identity-framework/pull/4074/files#r1004802482